NEWS.md

```diff
@@ -1,13 +1,45 @@
+# CRE Develop
+
+## Added
+* Add (vanilla) Stability Selection (without Error Control).
+* `max_rules` hyper parameters for max rules filtering.
+* Uncertainty Quantification in estimation by bootstrapping.
+* `B` hyper-parameter,
+* `subsample` hyper-parameter.
+* `rules`(implicit form) in cre() function return.
+* predict() function for ITE estimation via CRE.
+
+## Changed 
+* Type `stability_selection` binary -> string ('no','vanilla','error_control').
+* Unify `ntrees_gbm` hyper-parameter and `ntrees_gbm` hyper-parameter in 
+`ntrees` hyper-parameter.
+* In rules generation retrieve decision rules also from internal nodes, and not
+just from terminal nodes.
+* `ite_method_dis`, `ite_method_inf` method-parameter -> `ite_method`.
```

Comment:
Please also update the readme file. It is still based on `ite_method_dis` and `ite_method_inf`.